handlers/command_handler.py

```diff
@@ -142,4 +145,5 @@ def _delete_first_two_words(self, phrase: str) -> str:
             3: discord.Colour.gold(),
             4: discord.Colour.dark_red()
         }[worst_status]
-        await self.client.send_message(message.channel, embed=embed)
+        await self.client.send_message(discord.Object(id='347411900864135189'), mention_user)
```

Comment:
You might be misunderstanding send_message. 
send_message(destination, contents)

The old statement should suffice unless you want to PM the status to the user (see above for PMing the user)

Reply:
Like above `discord.Object(id='Channel ID')` sends a message to the channel stated in the parentheses. DM'ing is also an option, however, keeping the messages to a channel in the discord allows users to run commands on behalf of others that don't know they exist.